spring-boot-project/spring-boot-actuator/src/test/java/org/springframework/boot/actuate/neo4j/Neo4jHealthIndicatorTests.java

```diff
@@ -77,9 +78,8 @@ public void neo4jUp() {
 
 	@Test
 	public void neo4jDown() {
-		CypherException cypherException = new CypherException("Error executing Cypher",
-				"Neo.ClientError.Statement.SyntaxError",
-				"Unable to execute invalid Cypher");
+		CypherException cypherException = new CypherException(
```

Comment:
I welcome such polish but I'd prefer to see them in a separate commit.

Reply:
Was this resolved really? I can still see the change

Reply:
There are seven commits right now so it's hard for me to figure out what is supposed to be squashed and what isn't.

Generally speaking, if you have something that's good to go, I very much prefer a quick PR that I can merge right away. If you don't have time to create one, that's fine as well but please squash the current commits to make that a bit more explicit. Thanks!